src/app/lib/views/movie_detail.js

```diff
@@ -107,7 +107,7 @@
                 coverCache = null;
             };
             coverCache.onerror = function () {
-                $('.mcover-image').attr('src', self.model.get('image')).addClass('fadein');
+                $('.mcover-image').attr('src', this.model.get('image')).addClass('fadein');
```

Comment:
doesn't this change the context ? `this` is linked to onerror(), not to onShow()